app/views/home/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="container">
+  <div class="container my-3 d-flex justify-content-center">
+    <div>
+      <h1>Welcome to My Budget App</h1>
+      <p>Manage your budget without stress</p>
+    </div>
+  </div>
+  <div class="container my-3 d-flex justify-content-center">
+    <%= link_to "Login", new_user_session_path, class: "btn btn-outline-info me-2" %>
+    <%= link_to "Register", new_user_registration_path, class: "btn btn-outline-info" %>
+  </div>
+</div>
```

Comment:
- [ ] It is a requirement that we follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:

* Colors
* Typography: font face, size, and weight.
* Layout: composition and space between elements.

#### Overall your application does NOT follow this requirement. Kindly do so remember the font family and colors especially

Reply:
Thank you for pointing those out @Munsa1 I will do my best and correct them now